pkg/asset/cluster/tfvars.go

```diff
@@ -399,6 +404,11 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			Data:     data,
 		})
 	case baremetal.Name:
+		ignitionURL := &url.URL{
+			Scheme: "https",
+			Host:   net.JoinHostPort(installConfig.Config.Platform.BareMetal.APIVIP, "22623"),
+			Path:   fmt.Sprintf("/config/%s", "master"),
```

Comment:
Nit, could this just be `/config/master`?

Reply:
Yes. Easier that way. Will change.